src/main/java/appeng/util/Pair.java

```diff
@@ -0,0 +1,108 @@
+package appeng.util;
```

Comment:
There is javafx.util.Pair which is a part of JRE/JDK 8

Reply:
my dev running environment crashed when using javafx.util.Pair
```
Caused by: java.lang.NoClassDefFoundError: javafx/util/Pair

	at codechicken.nei.ItemPanel.handleClickExt(ItemPanel.java:323) ~[ItemPanel.class:?]
	at codechicken.nei.LayoutManager.mouseClicked(LayoutManager.java:171) ~[LayoutManager.class:?]
	at codechicken.nei.guihook.GuiContainerManager.mouseClicked(GuiContainerManager.java:368) ~[GuiContainerManager.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.mouseClicked(GuiContainer.java) ~[GuiContainer.class:?]
	at appeng.client.gui.AEBaseGui.mouseClicked(AEBaseGui.java:340) ~[AEBaseGui.class:?]
	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:296) ~[GuiScreen.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.handleMouseInput(GuiContainer.java) ~[GuiContainer.class:?]
	at appeng.client.gui.AEBaseGui.handleMouseInput(AEBaseGui.java:715) ~[AEBaseGui.class:?]
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:268) ~[GuiScreen.class:?]
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1640) ~[Minecraft.class:?]
	... 11 more
```

Reply:
weird, looks like JavaFX is not part of newer JDKs (version > 9)

Reply:
maybe ```import akka.japi.Pair;``` ?

Reply:
Is it "more standard" than JavaFX? or it is used my minecraft itself?

Reply:
akka's pair works in my dev and real game environment. but bartworks' pair is also written by itself so i am not sure whether we should use it.

Reply:
We should probably remove it too?

Reply:
then let's try to use akka'a pair. at least it doesn't crash or behave weird in my machine